### ISSUE TITLE
change mobile menu toggler text to sr-only when it's hidden

### ISSUE THREAD

**Issue by zebapy** [middlebury/frontend#188]
https://middlebury.slack.com/archives/G1KKMK4KF/p1581003598036600
